src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpExporterOptions.cs

```diff
@@ -95,7 +95,7 @@ public OtlpExporterOptions()
         /// <summary>
         /// Gets or sets the max waiting time (in milliseconds) for the backend to process each span batch. The default value is 10000.
         /// </summary>
-        public int TimeoutMilliseconds { get; set; } = 10000;
+        public uint TimeoutMilliseconds { get; set; } = 10000;
```

Comment:
This would be a breaking change.

Reply:
By convention all the .NET APIs use int as timeoutMs https://docs.microsoft.com/en-us/dotnet/api/system.threading.waithandle.waitone?view=net-5.0#System_Threading_WaitHandle_WaitOne_System_Int32_System_Boolean_.

Reply:
Oops I forgot this branch was still active in a PR and while making sure I had everything from my desktop pushed before switching to my laptop for travel.
The last commits were just as the commit message says ideas 😅 I'll clean this up and make the suggested changes in the above comment as well.